src/gentropy/config.py

```diff
@@ -30,13 +31,21 @@ class StepConfig:
     )
 
 
+class ColocalisationMethod(Enum):
```

Comment:
You should be asking the Colocalisation method module what's available and create an enum based on that. No need for a class.